content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -123,6 +125,33 @@ your cluster. Those fields are:
   See [Label Selectors](/docs/concepts/overview/working-with-objects/labels/#label-selectors)
   for more details.
 
+- **nodeAffinityPolicy** indicates how we will treat Pod's nodeAffinity/nodeSelector
+when calculating pod topology spread skew. Options are:
+  - Honor: only nodes matching nodeAffinity/nodeSelector are included in the calculations.
+  - Ignore: nodeAffinity/nodeSelector are ignored. All nodes are included in the calculations.
+
+  If this value is nil, the behavior is equivalent to the Honor policy.
```

Comment:
```suggestion
  If this value is null, the behavior is equivalent to the Honor policy.
```

In documentation, we use the values as if it was written in json/yaml.

Reply:
Yeah.